### ISSUE TITLE
GodMode9 doesn't find any games on SD NAND

### ISSUE THREAD

**Issue by niansa** [d0k3/GodMode9#436]
GodMode9 simply doesn't find any games on SD NAND. Just 2 times: "title.tmd".
New Nintendo 3DS XL v11.8.0-41E, Luma3DS 9.1, GodMode9 v1.7.1, Fastboot3DS v1.1

**Comment by d0k3** [comment]
@niansa - don't want to nag, but we need more info to resolve this. The issue is not reproducable for me.

**Comment by niansa** [comment]
Yes I mean A:/ drive.

 1. I go to A:/title
 2. I press R+A
 3. I select "Search for titles"
 4. It says "Found 2 results."
 5. I press A
 6. It shows a list of the following 2 files: "title.tmd" and "title.tmd". Both files are 520 bytes big and I can't get title infos aubout them or their .app files. The first one is inside A:/title/00030015/534c524e/content and the second one is inside A/title/00030015/53524c41/content.

I hope that was enough info. 😃

**Comment by d0k3** [comment]
Okay, can you maybe add screenshots (that's L+R)? One from step 1. (I want to see the contents of that folder) and one from step 6.

**Comment by d0k3** [comment]
Okay, some more info. @niansa - soemthing is very weird there. These two title IDs shouldn't be installed to the SD (the 'A:/' drive). These are TWL titles, and they should be installed to TWL (the '1:/'  drive). 

That means, there got to be some other mistake. Either you did something wrong when testing, or you somehow borked your SysNAND (maybe by manually copying and deleting stuff inside the `0:/Nintendo 3DS` folder?). I'll need your help trying to figure this out.

**Comment by niansa** [comment]
Fixed by deleting the A/title/00030015 folder. Thank you anyways.

**Comment by d0k3** [comment]
Reopening this issue. I'll look into a workaround. Still, something went very wrong here. Did you do anything special to your `0:/Nintendo 3DS` folder?

**Comment by d0k3** [comment]
Okay, @MYBABYODALYS - I need more info and your help. Can you make some screenshots to show me the contents of your `A:/` drive. Also, the contents of the `A:/title/00030015` folder (zipped and uploaded somewhere) would help.

**Comment by niansa** [comment]
Yes i copied TwiLightMenu++ title directory to the 3DS title directory. Was that wrong?

**Comment by d0k3** [comment]
@niansa - normally that should cause no harm. Neither for GM9, nor for homemenu. But, there is no use to it. TwiLightMenu++ is TWL software, it runs in DSi mode, and it has to stay in the TWL partition. Sooo... is there a reason why you did it?

**Comment by niansa** [comment]
I thought it belongs to the SD NAND.

**Comment by legoj15** [comment]
Actually, those are only meant for the DSi, not the 3DS SD card.

**Comment by d0k3** [comment]
@niansa - I'm still a bit clueless. If you copied the dir from TWLN (2:/), then there should be `00000000.tmd` file, not a `title.tmd` file. Also, your tmd file were too small (520 byte). This can't be a proper tmd file. So, from where did you copy these files?

@MYBABYODALYS - did you do the same thing @niansa did? If so, why? Also, note my other posts, I still need your help to get behind this.

**Comment by d0k3** [comment]
Okay, so what you guys did is, you tried to install TWiLightMenu++, didn't read the instructions, and you did it wrong. I still don't know exactly what you guys did, cause even with a wrong install the GM9 title search should still work.

I need more info.

@niansa - if you press R+A on the `00030015` folder, you only get the titles that are inside that folder. You're supposed to press R+A on `A:/` or on the `title` directory inside `A:/`. Is that what happened there?

**Comment by niansa** [comment]
@d0k3 "@niansa - if you press R+A on the 00030015 folder, you only get the titles that are inside that folder. You're supposed to press R+A on A:/ or on the title directory inside A:/. Is that what happened there?" No I did press that combination inside A:/titles folder. So this issue can be reopend!

**Comment by d0k3** [comment]
Yeah, you pressed it inside, @niansa. What you should have done is, point the cursor to the titles folder or the A: drive (ie. the object that you want to manipulate / search), then press R+A search for titles. That is, from inside the root dir or inside the A: drive. Not inside A:/titles.
